pkg/daemon/ceph/osd/volume.go

```diff
@@ -289,6 +306,14 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 	return nil
 }
 
+func getDatabaseSize(globalSize int, deviceSize int) int {
+	dbSize := globalSize
```

Comment:
Why not just using `globalSize`, why do you need a new variable? Also, we need test for this function.